plugin/neo-tree.vim

```diff
@@ -13,3 +13,6 @@ command! -nargs=? NeoTreeFloatToggle  lua require("neo-tree").float("<args>", tr
 command! -nargs=? NeoTreeFocusToggle  lua require("neo-tree").focus("<args>", true, true)
 command! -nargs=? NeoTreeShowToggle   lua require("neo-tree").show("<args>", true, true, true)
 command! NeoTreeRevealToggle lua require("neo-tree.sources.filesystem").reveal_current_file(true)
+
+set nospell
```

Comment:
This is not the place to set these. I'm pretty sure doing so here would affect the user's entire session. The places to apply the settings would be in the lua files at neo-tree.ui.renderer.create_window() and neo-tree.ui.popup.

Also, PRs go to main instaead of the v1.x release branch.